### ISSUE TITLE
avoiding PIN safety 

### ISSUE THREAD

**Issue by new-issue** [MiniKeePass/MiniKeePass#498]
Testet under iOS 11.0 beta:

1. install MiniKeePass
2. import database
3. activate PIN
4. activate database password storage
5. open database with password

To avoid PIN safety:
1. delete MiniKeePass
2. install MiniKeePass
3. import the same database es before
Now you have access to the database without entering the PIN

**Comment by tallerthenyou** [comment]
Great catch. I just tested this using the store version and I'm seeing the same behavior.
I'm having trouble replicating it with our newer version in Xcode, so I'm going to leave this open for testing with TestFlight

**Comment by tssmith7** [comment]
This is an iOS issue I think. See [here](https://forums.developer.apple.com/thread/36442).

**Comment by sumodx** [comment]
Yup, looks like iOS behaviour change. Mitigation advice from "Quinn “The Eskimo!” " also shows up in that thread:

> Third, my specific advice:
> If you want a keychain item to be deleted when your app is deleted, entangle it with a random key you store on disk, as described in my earlier post (11 Feb 2016).  This is guaranteed to work regardless of how the OS behaves.

**Comment by jamisonbennett** [comment]
I have a pull request under review here https://github.com/MiniKeePass/MiniKeePass/pull/636
 that addresses this.
